extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -226,12 +308,21 @@ export class RealTimeConfigManager {
             this.buildErrorResponse_(
                 RTC_ERROR_ENUM.UNKNOWN_VENDOR, vendor, errorReportingUrl));
       }
+      // There are two valid configurations of the vendor object.
+      // It can either be an object of macros mapping string to string,
+      // or it can be an object with sub-objects, one of which can be
+      // 'macros'. This is for backwards compatability.
+      const vendorMacros =
+            this.rtcConfig_.vendors[vendor]['macros'] &&
+            isObject(this.rtcConfig_.vendors[vendor]['macros']) ?
```

Comment:
Do you need the first check here?  I would assume isObject returns false if undefined.